sky/compositor/picture_layer.cc

```diff
@@ -18,10 +26,30 @@ void PictureLayer::Paint(PaintContext::ScopedFrame& frame) {
   DCHECK(picture_);
 
   SkCanvas& canvas = frame.canvas();
-  canvas.save();
-  canvas.translate(offset_.x(), offset_.y());
-  canvas.drawPicture(picture_.get());
-  canvas.restore();
+
+#if ENABLE_RASTER_CACHE
+  const SkMatrix& ctm = canvas.getTotalMatrix();
+  SkISize size = SkISize::Make(paint_bounds().width() * ctm.getScaleX(),
```

Comment:
The rasterization isn't 1-to-1 with screen pixels.  It's giving a pixel grid for the picture in its own coordinate system.  The rasterized image is transformed into screen space when you draw the image into the screen.  The goal is just to get the right order of magnitude of pixels so you don't get aliasing.  (Chrome actually rasterizes at two different scales, so it's somewhat arbitrary.)